ocm/samples/sample-manifest.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: component.kyma-project.io/v1alpha1
+kind: Manifest
+metadata:
+  labels:
+    operator.kyma-project.io/channel: stable
+    operator.kyma-project.io/controller-name: manifest
+    operator.kyma-project.io/kyma-name: kyma-sample
+  name: manifestkyma-sample
+  namespace: default
+spec:
+  # # crds are optional for watcher
+  # crds:
+  #   name: example.org/sample-module
+  #   ref: sha256:f6026672811a3b6ee3a222b9eefc4b51db46812bc01cf53aa971f1b10b72a25a
+  #   repo: k3d-registry.localhost:4000/component-descriptors
+  #   type: oci-ref
+  # # resource is optional for watcher
+  # resource:
+  #   kind: SampleCRD
+  #   resource: samplecrds
+  #   apiVersion: component.kyma-project.io/v1alpha1
+  #   metadata:
+  #     name: sample-crd-from-manifest
+  #     namespace: default
+  #   spec:
+  #     randomkey: samplevalue
+  config:
+    ref: sha256:27e42d46ea910ea5e49872ac88003d9d816eee416bb1f08a73a68c4b050619ce
+    name: example.org/sample-module
+    repo: k3d-registry.localhost:4000/component-descriptors
+    type: oci-ref
+  installs:
+    - source:
+        name: example.org/sample-module
+        repo: k3d-registry.localhost:4000/component-descriptors
+        ref: sha256:c1582fbe0241572ccf108221ec317bb3f4ef93549f5551f9e40b80cafe305a3a
+        type: oci-ref
+      name: watcher-chart # same name as chart for which the config will be applied
```

Comment:
this sample file should not be a part of this repo, but rather should be taken from the manifest-operator for consistency reasons